src/modules/graphics.wren

```diff
@@ -13,6 +13,8 @@ class Canvas {
       Fiber.abort("Default font must be a font name")
     }
   }
+  
+  static font {__defaultFont}
```

Comment:
Slight style nitpick, I prefer to have spaces between braces and the statement within.

This may also need an update to the documentation to make it clear this field can be both written to and read.